packages/config/README.md

```diff
@@ -0,0 +1,126 @@
+# Jetpack Configuration
+
+Allows for enabling and initializing of Jetpack features provided by
+other packages.
+
+# Usage
+
+Add this package as a dependency to your project:
+
+```
+composer require automattic/jetpack-config
+```
+
+Add every other package you're planning to configure:
+
+```
+composer require automattic/jetpack-sync
+composer require automattic/jetpack-tracking
+composer require automattic/jetpack-terms-of-service
+```
+
+In your code initialize the configuration package at or before
+plugins_loaded priority 9:
```

Comment:
I think the consumer needs to initialize the config package and call `Config::ensure()` at plugins_loaded priority 1. `Config::on_plugins_loaded` runs at priority 2.

Reply:
Yes, you're right, I forgot to update this part, thank you!